libtransmission/announcer.c

```diff
@@ -130,20 +135,43 @@ static int compareStops(void const* va, void const* vb)
         return i;
     }
 
-    /* tertiary key: the tracker's announec url */
+    /* tertiary key: the tracker's announce url */
     return tr_strcmp0(a->url, b->url);
 }
 
 /***
 ****
 ***/
 
+struct tr_scrape_info
+{
+    char * url;
+
+    int multiscrape_max;
+};
+
+static void scrapeInfoFree(void *va)
+{
+    struct tr_scrape_info* a = va;
+
+    tr_free(a->url);
+    tr_free(a);
+}
+
+static int compareScrapeInfo(const void* va, const void* vb)
```

Comment:
Here and everywhere: Once upon a time I've changed all the code to put `const` and `volatile` on the right of the type. I'd like to enforce this but https://github.com/uncrustify/uncrustify/issues/1952 is not there yet so I'm kindly asking to keep the style consistent.

Reply:
Agreed. Fixed eacd6e22046dde897af7b52459acc4bb2a6688c3